src/panels/config/cloud/login/cloud-login.ts

```diff
@@ -163,6 +154,15 @@ export class CloudLogin extends LitElement {
                   )}</ha-progress-button
                 >
               </div>
+              <button
```

Comment:
This is now outside of `.card-actions`, have you tested this PR?

Reply:
No I hadn't and didn't check that box when submitting the PR. I now have a proper dev environment set-up after many hours of trial & error and delighted to say that I have this working now, which is amazing! Sorry if you felt I wasted your time here with this, it was not my intent.